### DISCUSSION TITLE
Remove verification from Discord to foster collaboration

### DISCUSSION THREAD

**Discussion by keinproblem** [GameServerManagers/LinuxGSM#4259] [General]
By removing the verification check in Discord, overall collaboration would be strengthened.
There are some sitations where verification of Discord accounts might not be possible or convenient.

Proposed options:
- Remove verification
- Create a space for unverified contributors
- Use different verification schemes (that do not rely on phone number verification), such as Steam or GitHub

Alternatively, there still is good old IRC

**Comment by dgibbs64** [comment]
Phone number verification is a standard feature Discord provides to prevent spam and so far that has worked well. Discord does not offer verification via Steam or GitHub (though it might be a good feature to suggest to them). You are the first person to ever report an issue with this since we started using Discord a number of years ago. Since you have a GitHub account you are more than welcome to post issues/discussions here.

**Comment by keinproblem** [comment]
> Phone number verification is a standard feature Discord provides to prevent spam and so far that has worked well. Discord does not offer verification via Steam or GitHub (though it might be a good feature to suggest to them). You are the first person to ever report an issue with this since we started using Discord a number of years ago. Since you have a GitHub account you are more than welcome to post issues/discussions here.

Thanks, Daniel - I do appreciate your reply.

I do believe most people do not have any issues with verifying their acocunts via mobile phone and I do agree that this is a great feature with preventing spam and other types of malicious activities.

Indeed, it seems like raising a feature requests towards extending the regular verification procedure dicord implements currently makes more sens overall. This would allow service consumers, such as the LinuxGSM project, to rely on vanilla features.

What I mean with utilizing Steam or GitHub Auth is using a separate 3rd pt. bot that administers verification status. It would allow users to be trustworthy by providing Steam or GitHub with the authorization to present user data to the bot.
Utilizing 3rd party tools might be a step backwards here from a pure conceputal perspective. Considering that, I would not vote for such an option anymore.

I will probably rely on read-only Discord feeds and do collaboration within the GitHub project respectively.
